packages/build-info/src/bin.ts

```diff
@@ -1,31 +1,30 @@
-#!/usr/bin/env node
 import { exit, argv } from 'process'
 
+import yargs, { Arguments } from 'yargs'
```

Comment:
This is the entrypoint you should use with ESModules according to the documentation:

![CleanShot 2022-10-21 at 11 34 02](https://user-images.githubusercontent.com/11156362/197164534-a78ef709-3a14-4f67-b9e8-1a45e076bed2.png)

Which has a slightly different api on the `.command` https://github.com/yargs/yargs